modules/ROOT/pages/authentication-authorization/property-based-access-control.adoc

```diff
@@ -0,0 +1,58 @@
+:description: How to use Cypher to manage property-based access control on graphs.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE regularUsers;
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[property-based-access-control]]
+= Property-based access control
+
+It is possible to create `READ` privileges that are based on properties on nodes.
+To specify which property to base the `READ` privilege the `pattern` syntax described bellow is used,
+for more information about `READ` privilege syntax see xref:authentication-authorization/privileges-reads.adoc[read privilege] page.
+Adding property-based access control can incur a significant performance overhead in certain scenarios,
+see xref:authentication-authorization/limitations.adoc#property-based-access-control-limitations[Limitations] for more detailed information.
+To reduce the performance impact, we recommend using Block Storage as this is better optimised for the kind of read required for the resolution of property-based privileges.
+
+Pattern syntax:
+[source, syntax, role="noheader"]
+----
+([var][:label["|" ...]] "{" property: value "}")
```

Comment:
might be good to somewhere in the text explicitly state that we only allow property rules for nodes and a single property

Reply:
Could not come up with a good way of explaining this in the text. Added a note under the description of the pattern syntax. What do you think?

Reply:
Could work, what about something like https://github.com/neo4j/docs-operations/pull/1076#discussion_r1357853921